[MIG] portal_multicurrency_totals: Migration to 17.0 t#79454

CLaurelB opened the pull request.

luisg123v committed:
[ADD] portal_multicurrency_totals: Show multicurrency totals on portal

This module adds te feature to display total amounts by currency when
accessing documents through the portal:
- Quotations
- Sale orders
- Invoices

desdelinux commented:
Hi @hugho-ad Please review.

luisg123v commented on portal_multicurrency_totals/README.rst:
```diff
@@ -16,13 +16,13 @@ To use this module, just go to the portal  and open a document list; you will se
 currency:
 
-    .. image:: portal_multicurrency_totals/static/description/totals_in_so.png
+    .. image:: /portal_multicurrency_totals/static/description/totals_in_so.png
```
Why this change?

Please check if this image is being rendered correctly on GitHub.

CLaurelB replied:
I changed it because the image was being shown as a broken link in the previous version.

luisg123v commented on portal_multicurrency_totals/demo/res_partner_demo.xml:
```diff
@@ -2,24 +2,21 @@
+            name="user_id"
             model="res.partner"
             eval="[
             (0, 0, {
```
Migrate to the command syntax.

And let's take the chance to fix indentation here.

CLaurelB replied:
Fixed.

luisg123v commented on portal_multicurrency_totals/demo/res_partner_demo.xml:
```diff
@@ -35,10 +32,6 @@
-        name="change_password_button"
-        eval="[ref('set_password_portal_user')]"
-    />
+    <function model="change.password.wizard" name="change_password_button" eval="[ref('set_password_portal_user')]" />
```
Perform autoformatting changes on a separate commit located before this one, as we would do when migratin an OCA module.

CLaurelB replied:
Done.

luisg123v replied:
Please include in the history all commits merged in v14 (#1648).

CLaurelB replied:
Done.

CLaurelB committed:
[FIX] portal_multicurrency_totals: fix README usage images reference path

This update corrects the path to the usage images in the README,
ensuring they are displayed properly.

CLaurelB committed:
[IMP] portal_multicurrency_totals: pre-commit-vauxoo auto fixes

Apply pre-commit-vauxoo auto fixes.

CLaurelB committed:
[FIX] portal_multicurrency_totals: fix indentation on demo data

Fix indentation on the wizard records.

CLaurelB committed:
[REF] portal_multicurrency_totals: fix xml headers

Modify the header of xml files having the encoding as UTF-8.

CLaurelB committed:
[MIG] portal_multicurrency_totals: Migration to 17.0

luisg123v reviewed (approved):
LGTM, but this is red.

Could you fix it in a separate PR, please?

Just cherry-pick c84e8fe24bffb47eeae5bbc9fa0c4bd111397d0b

CLaurelB commented:
> LGTM, but this is red.
> 
> Could you fix it in a separate PR, please?
> 
> Just cherry-pick [c84e8fe](https://github.com/Vauxoo/addons-vauxoo/commit/c84e8fe24bffb47eeae5bbc9fa0c4bd111397d0b)

Done in https://github.com/Vauxoo/addons-vauxoo/pull/1650

luisg123v commented:
@CLaurelB thanks, just rebase.

CLaurelB commented:
> @CLaurelB thanks, just rebase.

Done.
